File: apzkr-pzpi-21-11-kokhanevych-pavlo/Task1-Server/src/main/java/com/pet/hotel/businessLogic/domain/transferObjects/user/HistoryRentingResponse.java
package com.pet.hotel.businessLogic.domain.transferObjects.user;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class HistoryRentingResponse {

    private LocalDateTime beginDate;

    private LocalDateTime endDate;

    private String petName;

    private int roomId;

    private int rentId;

    private int numberRoom;

    private String hotelName;

    private int totalPayment;
}

File: apzkr-pzpi-21-11-kokhanevych-pavlo/Task4-MobileClient/src/main/java/com/animal/hotel/domain/models/user/HistoryRentingResponse.kt
package com.animal.hotel.domain.models.user

import java.time.LocalDateTime

data class HistoryRentingResponse(
    val beginDate: LocalDateTime,
    val endDate: LocalDateTime,
    val petName: String,
    val roomId: Int,
    val rentId: Int,
    val numberRoom: Int,
    val hotelName: String,
    val totalPayment: Int
)
